qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -197,12 +202,31 @@ def result(self, timeout: Optional[float] = None, wait: float = 5) -> Result:
             results again in another instance or session might fail due to the
             job having been consumed.
 
+
+            When `partials=True`, the result method may return a `Result` object
```

Comment:
Small tweaks:
* probably `partial=True` instead of `partials`?
* can you have just one blank line at the top? 
* also, can you double-check if after the latest changes (exception instead of result) this sentence continues being accurate - specifically the "may" (it probably is fine - just wondering)?

> When `partials=True`, the result method **may** return a `Result` object